docs/en/operation/StressBench.md

```diff
@@ -111,3 +111,158 @@ This process continues until the max throughput is discovered.
+In the cluster setting, it uses Alluxio job service to mock the distributed training task with Alluxio POSIX API and measures the overall reading throughput as well as each individual client.
+
+### General usage
+By reading the self-generated test files through the Alluxio Fuse-based POSIX API, this stress bench is able to calculate the reading throughput configured with a certain setup of the cluster and the test. 
+
+
+#### Parameters
+The parameters for the Fuse IO Stress Bench are (other than common parameters for any stress bench):
+
+<table class="table table-striped">
+    <tr>
+        <td>Parameters</td>
+        <td>Default Value</td>
+        <td>Description</td>
+    </tr>
+    <tr>
+        <td>operation</td>
+        <td>Required. No default value.</td>
+        <td>The operation that is going to perform. Available operations are [Write, ListFile, LocalRead, RemoteRead, ClusterRead], where Write and ListFile are for testing reading performance, not individual tests.</td>
+    </tr>
+    <tr>
+        <td>local-path</td>
+        <td>/mnt/alluxio-fuse/fuse-io-bench</td>
+        <td>The path that the operation is going to perform at. Can be a local filesystem path or a mounted Fuse path.</td>
+    </tr>
+    <tr>
+        <td>threads</td>
+        <td>1</td>
+        <td>The number of concurrent threads that is going to be used for the operation.</td>
+    </tr>
+    <tr>
+        <td>num-dirs</td>
+        <td>1</td>
+        <td>The number of directories that the files will be evenly distributed into. It must be at least the number of threads and preferably a multiple of it. </td>
+    </tr>
+    <tr>
+        <td>num-files-per-dir</td>
+        <td>1000</td>
+        <td>The number of files per directory.</td>
+    </tr>
+    <tr>
+        <td>file-size</td>
+        <td>100k</td>
+        <td>The files size for IO operations. (100k, 1m, 1g, etc.)</td>
+    </tr>
+    <tr>
+        <td>warmup</td>
+        <td>15s</td>
+        <td>The length of time to warmup before recording measurements. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+    <tr>
+        <td>duration</td>
+        <td>30s</td>
+        <td>The length of time to run the benchmark. (1m, 10m, 60s, 10000ms, etc.)</td>
+    </tr>
+</table>
+
+### Testing steps
+#### Prerequisite
+* A running Alluxio cluster.
+* Mount the testing Alluxio path to a local file system path. See
+  [FUSE-based POSIX API](https://docs.alluxio.io/os/user/stable/en/api/POSIX-API.html) for more details on mounting.
+* (Optional) To prevent operating system cache and Fuse cache from affecting benchmark accuracy, using different mount points for writing and reading test files. 
+
+#### Write test files
+Write the test files by running the benchmark with `--operation Write` into the mount point for writing files. More specifically, run 
```

Comment:
usually `run` and code block are close to each other
same for `For example,` and code block